testing/nqp/APKBUILD

```diff
@@ -0,0 +1,50 @@
+# Contributor: Curt Tilmes <Curt.Tilmes@nasa.gov>
+# Maintainer: Curt Tilmes <Curt.Tilmes@nasa.gov>
+pkgname=nqp
+pkgver=2018.02
+pkgrel=0
+pkgdesc="Not Quite Perl"
+url="https://github.com/perl6/nqp"
+arch="all"
+options="!archcheck" # Arch dependencies are embedded
+license="Artistic-2.0"
+depends="moarvm"
+makedepends="perl-utils gcc musl-dev make moarvm-dev"
+install=""
+subpackages="$pkgname-doc $pkgname-examples"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/perl6/nqp/archive/${pkgver}.tar.gz"
+builddir="$srcdir"/"$pkgname"-"$pkgver"
+
+build() {
+	cd "$builddir"
+        perl Configure.pl --prefix=/usr --backends=moar
+        make
+}
+
+check() {
+        cd "$builddir"
+        make test
+}
+
+package() {
+	cd "$builddir"
+        make DESTDIR="$pkgdir" install
+}
+
+doc() {
+	cd "$builddir"
+	mkdir -p "$subpkgdir"/usr/share/doc/"$pkgname"
+	for file in CREDITS LICENSE README.pod VERSION docs; do
+                cp -r $file "$subpkgdir"/usr/share/doc/"$pkgname"
+        done
+}
+
+examples() {
+	cd "$builddir"
+        mkdir -p "$subpkgdir"/usr/share/doc/"$pkgname"
+        for file in examples; do
+                cp -r $file "$subpkgdir"/usr/share/doc/"$pkgname"
```

Comment:
Typically, this is installed to `"$subpkgdir"/usr/share/doc/"$pkgname"/examples`, and is installed with -doc.  However, if this is a large set of examples it may be better to use a separate subpackage.